src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write your code here
+function createDiv() {
+  const div = document.createElement('div');
+
+  return div;
+}
```

Comment:
why do we need this function, you can just create it directly document.createElement('div')